requirements.txt

```diff
@@ -1,3 +1,3 @@
-torch==1.8.1
-torchvision==0.9.1
+# torch==1.8.1
```

Comment:
Not compatible with torch v1.81?

Reply:
nope, since my developed environment is torch 1.9.0, I remove the requirement of PyTorch 1.8.1 temporarily. But this tengine expansion can be totally compatible with torch v1.8.1, and I will restore the requirements.